sounds/models.py

```diff
@@ -1049,7 +1050,7 @@ class Meta:
 
 
 class Download(models.Model):
-    user = models.ForeignKey(User)
+    user = models.ForeignKey(User, related_name='sound_downloads')
```

Comment:
Did you run `makemigrations? I think this changes generate migrations in Django that you should add in the PR.

Reply:
Good point! Included in next commit